example/90-shoot.yaml

```
@@ -55,6 +55,7 @@ spec:
     # kubeletDataVolumeName: kubelet-dir
     # providerConfig:
     #   <some-provider-specific-worker-config>
+    # allowSystemComponents: true
```

API-wise, such a structure would be more prepared in case we want to introduce further configuration settings for the system components in the future:

```suggestion
    # systemComponents:
    #   allowed: true
```

WDYT?

---

Sound reasonable to me, I'll change it

---

Done